assets/training/model_management/src/run_model_preprocess.py

```diff
@@ -140,7 +141,16 @@ def run():
                 AzureMLError.create(UnsupportedTaskType, task_type=args.task_name,
                                     supported_tasks=list(supported_tasks))
             )
-
+    files = check_for_py_files(model_path)
+    logger.info(f"check if model folder contains .py files or not: {files}")
+    if files:
+        if hf_model_args is None:
```

Comment:
This check is not sufficient. Add check if hf_model_args does not contain remote_trust_code=true

Reply:
updated with required checks